CMakeLists.txt

```diff
@@ -86,8 +86,11 @@ option(FIREBASE_GITHUB_ACTION_BUILD
 option(FIREBASE_QUICK_TEST
        "Enable quick tests will skip tests which requires access to the SECRET" OFF)
 
-set(FIREBASE_PYTHON_EXECUTABLE "python" CACHE FILEPATH
-  "The Python interpreter to use, such as one from a venv")
+find_program(FIREBASE_PYTHON_EXECUTABLE
```

Comment:
Can this still be overridden by specifying `-D FIREBASE_PYTHON_EXECUTABLE=xxx`? I depend on that functionality.

Reply:
Yep, documentation suggested that, and confirmed it locally.